tasks/libs/releasing/notes.py

```diff
@@ -42,7 +43,7 @@ def _add_dca_prelude(ctx, agent7_version, agent6_version=""):
             f"""prelude:
     |
     Released on: {date.today()}
-    Pinned to datadog-agent v{agent7_version}: `CHANGELOG <https://github.com/{GITHUB_REPO_NAME}/blob/{DEFAULT_BRANCH}/CHANGELOG.rst#{agent7_version.replace('.', '')}{agent6_version}>`_."""
+    Pinned to datadog-agent v{agent7_version}: `CHANGELOG <https://github.com/{GITHUB_REPO_NAME}/blob/{get_default_branch()}/CHANGELOG.rst#{agent7_version.replace('.', '')}{agent6_version}>`_."""
```

Comment:
not related to this PR but we might need to adapt this if we want it to work properly for both agent6 & 7: something that could look like
```suggestion
    Pinned to datadog-agent v{agent_version}: `CHANGELOG <https://github.com/{GITHUB_REPO_NAME}/blob/{get_default_branch()}/CHANGELOG.rst#{agent_version.replace('.', '')}>`_."""
```